src/commands/synthetics/reporters/default.ts

```diff
@@ -441,14 +441,14 @@ export class DefaultReporter implements MainReporter {
     test: Pick<Test, 'name'>,
     testId: string,
     executionRule: ExecutionRule,
-    config: UserConfigOverride
+    testOverrides: UserConfigOverride
```

Comment:
This file is not publicly exported so changing the name here is not a breaking change